contracts/SLA.sol

```diff
@@ -26,6 +26,11 @@ contract SLA is Staking {
         Status status;
     }
 
+    struct Governance {
+        uint64 leverage;
+        uint64 cap;
+    }
```

Comment:
This is clashing with our existing implementation of governance in `setStakingParameters`.
`leverage` and `cap` are also different in nature: one is a user input that makes sense as an argument, the other is a technical limitation that is never user-facing.